src/script/page/LeftSidebar/panels/Conversations/FilteredGroupConversations/FilteredGroupConversations.tsx

```diff
@@ -86,14 +86,18 @@ export const FilteredGroupConversations = ({
   }
 
   return (
-    <div>
+    <>
       <h3 css={headingTitle}>{t('searchGroups')}</h3>
 
-      <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name="conversation-view">
+      <ul
+        css={css({margin: 0, paddingLeft: 0})}
+        data-uie-name="filtered-conversation-view"
+        className="filtered-conversations"
```

Comment:
As you added a css class only for this `<ul>`, would be nice if we move the css there too.